src/implementations/vanilla/components/blockquote/_blockquote.scss

```diff
@@ -49,52 +51,61 @@ $_text-max-width: 65ch; // Approximately 80 characters with the current font
   }
 }
 
-.ecl-blockquote__attribution {
+.ecl-blockquote__attribution,
+%ecl-blockquote-author {
```

Comment:
You didn't use a "BEM" syntax here to avoid the `percent-placeholder-pattern` error that you were getting..? I pushed a configuration of stylelint so that we don't need to disable the checks by our BEM syntax plugin in our components but it would also be possible to do a different thing: adapting the current checks by our plugin to also check the selectors with the placeholder for their "BEM" syntax and disable the `percent-placeholder-pattern`. This is how i initially did this, then i noticed that your selectors were not "bem" compliant, so i pushed  just an ignore for our plugin.

Reply:
the reason why I use placeholders is because that's highly recommended to do so when using extend. It prevent unwanted side effects and potential duplications in compiled CSS.
see for instance:
- https://webinista.com/updates/dont-use-extend-sass/
- https://stackoverflow.com/questions/54084022/what-does-extend-must-be-used-with-a-placeholder-mean

Reply:
Yeah, ok, fine for the placeholders, still i would use the same convention that we use for the classes, so BEM, if we decide so, i will make the needed changes in the eslint configuration for that not to make the lint check fail.

Reply:
the linter fails even with BEM conventions (it just doesn't like the % thingy...).
but indeed, I will update the placeholder names

Reply:
it seems like BEM patterns like '__' or '--' are not valid for placeholders by default.
We could override the linter configuration, but I guess it is cleaner to just follow the default rule here and keep current naming

Reply:
but even normal selectors (classes) using BEM would not be "allowed" in the default configuration of stylelint we have, we do this to prevent the linter from complaining:     
```
// Allow underscores in class names (BEM)
    'selector-class-pattern': null,
```
and we have our own plugin to check the compliance of the BEM selectors we define.
So, i think for consistency, we can just treat the selectors the same way, whether they are classes or placeholders

Reply:
So, if we agree about this i will push the configuration for including the placeholders in the same check we do for the BEM syntax of our classes and i will disable the rule that checks the placeholders in stylelint.